index.rst

```diff
@@ -326,33 +326,162 @@ Analysis of unbinned data is clearly needed for pipeline data quality analyses,
-- The database must be available and rapidly synced to at all major data facilities (`FAFF-REQ-0055`_), analogous to what is done for the summit EFD.
-- Summit tooling, including the Scheduler, must have immediate access to the database (`FAFF-REQ-0056`_).
+The Engineering and Facilities Database (EFD)
+"""""""""""""""""""""""""""""""""""""""""""""
+
+The telemetry data from the observatory systems is recorded in the EFD.
+The data is organized by topics following the `T&S XML schema`_ and recorded in InfluxDB, a time series database optimized for queries constrained by time on large datasets.
+
+The EFD is managed by `Sasquatch`_, an RSP service that includes tools like Chronograf for data exploration and visualization and Kapacitor for alerting on the EFD data
+(see also :ref:`Metric databases in Sasquatch <metric-databases>`).
+
+.. _the-butler-registy:
+
+The Butler registry
+"""""""""""""""""""
+
+Rapid Analysis will use the Science Pipelines infrastructure for SFP.
+That infrastructure provides a Butler registry, a database for tracking processing input and output datasets.
+
+The current plan is to have a Butler repository per instrument at the Summit.
+
+Once the exposures are ingested into the Butler "raw" collection, IDs are assigned, and the registry becomes the primary source for obtaining the list of observations.
+
+The Butler registry schema is described in `DMTN-073`_.
+It is designed to support visits composed of multiple exposures.
+If the visit is composed of a single exposure, then the visit table has entries that are essentially duplicates of the science exposures in the exposure table.
+
+However, the registry database contains only a small subset of the raw image metadata (the information about the conditions under which an image was taken summarized in the image headers).
+
+Analysis outputs are not in the registry database; they are typically linked to the visit table in the registry and stored as files in the Butler datastore.
+An example of an SFP task that produces such ouput is the `ConsolidateVisitSummaryTask`_ task that persists per-detector summary statistics computed for calibrated exposures into a per-visit dataset in Parquet files.
+
+Expanding the Butler registry database to include the required raw image metadata and analysis outputs is outside the plans.
+Hence the need for a :ref:`Visit database <visit-database>` (see also `DMTN-227 The Consolidated Database of Image Metadata`_).
+
+.. _visit-database:
+
+The Visit database
+""""""""""""""""""
+
+The initial content for the Visit database is described in the database `use cases`_ confluence page and summarized in this section.
+We also discuss implementation details for creating and updating the Visit database tables.
+
+Similarly to the Butler registry, a PostgreSQL database is the natural choice for the Visit database implementation.
+The Visit database should be replicated at the USDF, and the Summit instance should keep data for at least 30 days.
```

Comment:
I think we said we don't want to say "replicated" 
Consider:
The visit database must be available at the summit and USDF. 
Again we leave the implementation (e.g. replication vs reprocessing) to the future developers but it is a strict requirement that the contents be identical in both instances.

Reply:
We still want to keep only 30 days at the Summit, so I would add that too:

```
Again we leave the implementation (e.g., replication vs. reprocessing) to future developers.
Still, it is a strict requirement that the contents be identical in both instances, with the nominal retention period of 30 days at the Summit.
```